addons/mail/models/ir_actions_server.py

```diff
@@ -179,9 +186,9 @@ def _check_activity_date_deadline_range(self):
     @api.constrains('state', 'model_id')
     def _check_model_coherency(self):
         for action in self:
-            if action.state in ('followers', 'next_activity') and action.model_id.transient:
+            if action.state in {'mail_post', 'followers', 'remove_followers', 'next_activity'} and action.model_id.transient:
                 raise ValidationError(_("This action cannot be done on transient models."))
-            if action.state == 'followers' and not action.model_id.is_mail_thread:
+            if action.state in {'followers', 'mail_post', 'remove_followers'} and not action.model_id.is_mail_thread:
                 raise ValidationError(_("Add Followers can only be done on a mail thread models"))
```

Comment:
Error message will be very confusing now that it accounts for more than "add followers" ^^